src/mlpack/core/data/map_policies/missing_policy.hpp

```diff
@@ -0,0 +1,108 @@
+/**
+ * @file missing_policy.hpp
+ * @author Keon Kim
+ *
+ * Missing map policy for dataset info.
+ */
+#ifndef MLPACK_CORE_DATA_MAP_POLICIES_MISSING_POLICY_HPP
+#define MLPACK_CORE_DATA_MAP_POLICIES_MISSING_POLICY_HPP
+
+#include <mlpack/core.hpp>
+#include <unordered_map>
+#include <boost/bimap.hpp>
+#include <mlpack/core/data/map_policies/datatype.hpp>
+#include <limits>
+
+
+using namespace std;
+
+namespace mlpack {
+namespace data {
+
+/**
+ * Same as increment map policy, but does not change type of features.
+ */
+class MissingPolicy
+{
+ public:
+  // typedef of mapped_type
+  using mapped_type = double;
```

Comment:
But a problem here is that the actual mapped type is specified by the input type to `data::Load()`, which takes matrices of any type.  So even if `mapped_type` (which should be named `MappedType` by the way) is `double`, if the user passed an `arma::Mat<size_t>` to `data::Load()`, any NaNs that were mapped will be converted to something else when the `double` is converted to a `size_t`.

So I think that you will have to take the type as given by `data::Load()`, and maybe then you can use a `static_assert()` to check that the given type has what you need (like for `MissingPolicy`, you need some kind of NaN).